### PR TITLE
glog: new package

### PR DESCRIPTION
**Pull request by amir-sabbaghi**
glog is C++ implementation of the Google logging module

Signed-off-by: Amir Sabbaghi <amir@pichak.co>

-------------------------------

Maintainer: me 
Compile tested: MIPS, GL-inet 6416 v1 and GL MT300A, LEDE commit 5faad30
Run tested: MIPS, GL-inet 6416 v1 and GL MT300A, LEDE commit 5faad30

### CODE REVIEW

**Comment by rotoroute** [comment]
Please merge, I wish use this package.

File: libs/libglog/Makefile [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  DEPENDS:= +libstdcpp +libpthread
+  URL:=https://github.com/google/glog
+  MAINTAINER:=Amir Sabbaghi <amir@pichak.co>
+endef
+
+define Package/glog/description
+  This repository contains a C++ implementation of the Google logging
+  module.  Documentation for the implementation is in doc/.
+endef
+
```
Comment (hauke) [inline libs/libglog/Makefile]: Please add TARGET_CXXFLAGS and TARGET_LDFLAGS here

File: libs/libglog/Makefile [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+PKG_NAME:=glog
+PKG_VERSION:=0.3.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/google/glog/archive
+PKG_MD5SUM:=df92e05c9d02504fb96674bc776a41cb
+
+PKG_FIXUP:=autoreconf
+PKG_USE_MIPS16:=0
```
Comment (hauke) [inline libs/libglog/Makefile]: What error message do you get with mips16? This is just a question, if you get one this is ok.

Comment (amir-sabbaghi) [inline libs/libglog/Makefile]: I don't really remember. I have only tested ramips and it's working without it. I can remove it if you insist.

Comment (amir-sabbaghi) [inline libs/libglog/Makefile]: I removed it anyway. We can add it when needed.

File: libs/libglog/Makefile (hunk 1 above)
Comment (amir-sabbaghi) [inline libs/libglog/Makefile]: Fixed.

File: libs/libglog/Makefile [hunk 3]
```diff
@@ -0,0 +1,53 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=glog
+PKG_VERSION:=0.3.5
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
```
Comment (champtar) [inline libs/libglog/Makefile]: This will eventually clash with other packages, best is to use `PKG_SOURCE_PROTO:=git`

Comment (amir-sabbaghi) [reply libs/libglog/Makefile]: Done

**Pull request merged**